qml/YubiKey.qml

```diff
@@ -239,7 +239,7 @@ Python {
                 if (!currentDevice || !availableDevices.some(dev => dev.serial === currentDevice.serial)) {
                     // new device is being loaded, clear any old device
                     clearCurrentDeviceAndEntries()
-
+                    navigator.goToLoading()
```

Comment:
Is this the spinning circle? Nice touch!

Reply:
It is, yes.